From 45da3b1bff79a50fc9c1343c5684282ab44667ad Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Mon, 17 Dec 2018 14:40:20 +0800
Subject: [PATCH] arm64: dts: rockchip: improve max eMMC clock to 200MHz for
 RK1808 evb

The IO capability for RK1808 was intent to be able work under 200MHz
clock, and the test result seems positive. However, we want a broader
test, so let's improve the max clock to 200MHz for all evb.

Change-Id: Ic7c694af92cb61a5d75edca7515f9f83f6ca98e4
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
index 8ca1802b624d..2ef387cd96e8 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
@@ -243,6 +243,7 @@
 &emmc {
 	bus-width = <8>;
 	cap-mmc-highspeed;
+	max-frequency = <200000000>;
 	mmc-hs200-1_8v;
 	supports-emmc;
 	non-removable;
-- 
2.35.3

